conformance/packages/dns-test/src/resolver.rs

```diff
@@ -130,24 +119,10 @@ impl ResolverSettings {
             container.cp(path, &contents)?;
         }
 
-        let mut child = container.spawn(implementation.cmd_args(config.role()))?;
-
-        // For HickoryDNS we need to wait until its start sequence finished. Only then the server is able
```

Comment:
@justahero I didn't see any test that required this startup wait so I removed it

was this added for some macOS specific issue?

it does seem inconsistent to only do this when using hickory because all the other DNS implementations are prone to the same problem  ...

Reply:
This logic might have been added because of the `terminate_x_works` tests inside the same file, where the `terminate` call immediately follows after the `Resolver` has started. I think the issue was that the `hickory-dns` created a pid file before the start sequence was fully executed (reaching [this line](https://github.com/hickory-dns/hickory-dns/blob/main/bin/src/hickory-dns.rs#L578)), maybe causing an issue during shut down. The previous logic might have been sufficient.

> it does seem inconsistent to only do this when using hickory because all the other DNS implementations are prone to the same problem ...

Sure, these implementation manage their pid file.

If that logic is unnecessary the better, the logs then contain the full start sequence output, similar to `unbound` & `bind` I guess.